integreat_cms/cms/views/pages/page_form_view.py

```diff
@@ -254,6 +254,21 @@ def post(self, request, *args, **kwargs):
                 page_translation_form.instance.page = page_form.save()
             # Save page translation form
             page_translation_form.save()
+            # If any source translation changes to draft, set all depending translations/versions to draft
+            if (
+                "status" in page_translation_form.changed_data
+                and page_translation_form.cleaned_data.get("status") == status.DRAFT
+            ):
+                language_tree_node = region.language_tree_nodes.get(
+                    language_id=language.id
+                )
+                if not language_tree_node.is_leaf():
+                    PageTranslation.objects.filter(
+                        page_id=page_translation_form.instance.page_id,
+                        language_id__in=[
+                            node.id for node in language_tree_node.get_descendants()
+                        ],
```

Comment:
This does only work in some cases by accident - language ids and language tree node ids are typically not identical.